### ISSUE TITLE
The 'docker_compose' provisioner could not be found.

### ISSUE THREAD

**Issue by rajeevaroraau** [SLS-ALL/devops-microcosm#2]
There seems to be 'docker_compose' provisioner not found error towards the end.

It may be due to lack of "reachability" (network issues?) or accessibility to docker-compose in the staging server.

Any ideas?

**Comment by rajeevaroraau** [comment]
Resolved this by installing vagrant docker-compose plugin.  Self closed.
